lib/sparkampserver.py

```diff
@@ -98,6 +112,106 @@ def eject(self):
 
         self.connected = False
 
+    def send_preset(self, chain_preset):
+        gate_parameters = chain_preset.gate_pedal.parameters()
+        comp_parameters = chain_preset.comp_pedal.parameters()
+        drive_parameters = chain_preset.drive_pedal.parameters()
+        amp_parameters = chain_preset.amp_pedal.parameters()
+        mod_parameters = chain_preset.mod_pedal.parameters()
+        delay_parameters = chain_preset.delay_pedal.parameters()
+        reverb_parameters = chain_preset.reverb_pedal.parameters()
+
+        preset_json = {"Preset Number": [0x00, 0x7f],
+                       "UUID": "CDE99591-C05D-4AE0-9E34-EC4A81F3F84F",
```

Comment:
This should be the unique UUID from the ChainPreset record.

Reply:
This code now uses the UUID from the ChainPreset record. Tested and working as expected.